Ports/optipng/package.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="optipng"
+version="0.7.7"
```

Comment:
For consistency, we have recently started to single-quote strings that do not require variable expansion. (Yes, actually documenting and checking that is somewhere on the yakstack.)

Reply:
GitHub does a bad job at pointing out which lines were targeted in the inline comments, this was also meant to apply for `port` and `auth_type`.